app/views/webview/drug_stocks/new.html.erb

```diff
@@ -77,5 +74,16 @@
       </div>
     </div>
   </div>
+
   </body>
+
+  <script type="text/javascript" charset="utf-8">
+  window.addEventListener("load", function() {
+    monthSelect = document.getElementById("for_end_of_month");
+    monthSelect.addEventListener("change", function(e) {
+      window.location.href = location.origin + location.pathname + location.search + "&for_end_of_month=" + encodeURIComponent(monthSelect.value);
+    });
+  })
+  </script>
```

Comment:
@msasikanth Do you know if adding javascript here will require any changes to the Android side?

We could make this work without JS, it would just require some re-working of the date selection UI and wouldn't be as nice.